### ISSUE TITLE
Suggestion: Separate hunch deck for Joe Diamond

### ISSUE THREAD

**Issue by szwanger** [Kamalisk/arkhamdb#604]
It would be incredibly helpful if Joe Diamond decks were able to create their Hunch Deck by selecting 11 Insight cards (which are already listed separately as hunches).  Currently it lists all Insight cards as hunches even when the deck has more than 11, but since Joe is required to have exactly 11 cards in the Hunch Deck, there's no visual way to differentiate which cards are in which part of the deck.  It would make the process of building a Joe Diamond deck much faster, smoother, and more efficient.

**Comment by Hounk** [comment]
The Hunchdeck is not made at deck creation, but during the setup of each scenario. So people might want to change it depending on the scenario.

**Comment by szwanger** [comment]
Yes, I am aware the hunch deck is made at the beginning of the scenario.  That's all the more reason to differentiate it from the rest of the decklist on arkhamdb.  The decklist itself is changing from one scenario to another if it's being played in a campaign, so the user can update the hunch deck at the same time they update the decklist.

The advantages of differentiating the hunch deck are to help with deckbuilding (so it's easy to visualize what the hunch deck contains and doesn't require rechecking and recounting every time a change is made), to facilitate setup (so anyone can build the hunch deck, they don't have to wait for the player), and to save time (so the user isn't studying their cards during setup and trying to remember what they planned -- all they need to do is match the hunch deck's list).  And if the player wants to change it depending on the scenario, they still can do so.

**Comment by Hounk** [comment]
Sorry, I don't see the advantage there. The hunch deck consists always of the same number of cards, 11, including his weakness. So, if you have 14 Insight events, it does not require a Master degree in Mathematics to calculate, you have to remove 3 and shuffle them into the main deck. But you can pick and choose different cards for each game of the campaign, or you can take notes on the side, which you deside to leave out of the hunch deck each game. What the deck builder calls "Hunches", could also be called "Insight events", or "Cards, that are eligible for the Hunch deck", but I find it pretty conveniant, that it lists all of these cards in one place. Maybe others agree more with your point, but I don't see it.

**Comment by szwanger** [comment]
It may well be that you would choose not to use this feature.  That doesn't mean that other players wouldn't appreciate it -- just as there are other features that you may use that others don't care about.  I still believe it would be incredibly useful.

**Comment by Hounk** [comment]
But what you request is not an additional "take it or leave it" feature, it is a replacement, that imo would hamper the usabillity, because the way, it is now, that ALL eligible cards are marked as possible hunches has more value to me. I don't see, how there is an issues to "calculate" how many insights you have to place into your regular deck, given the fact, ArkhamDB even counts them for you. On the other hand it can be rather clunky to double check all events, whether they are an Insight or not, if you want to alter your hunch deck, if they are not marked that way.
